### ISSUE TITLE
Custom css cannot be edited in some browsers

### ISSUE THREAD

**Issue by luo-chuan** [RSSNext/Follow#1780]
### Clear and concise description of the problem

As figure shows, some BBC's article shows a white picture, that will bring worse experience in dark mode.
These feed urls have this problem:
`https://plink.anyfeeder.com/bbc/world`
`https://plink.anyfeeder.com/bbc/technology`
The white picture's src is `https://www.bbc.com/bbcx/grey-placeholder.png`
![Image](https://github.com/user-attachments/assets/9882a9c6-d4da-4d00-ab62-93be6936d383)

### Suggested solution

Pls allow user to customize CSS, then user can add CSS like this:
```
img[src="https://www.bbc.com/bbcx/grey-placeholder.png"] { display: none; }
```

### Validations

- [x] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [x] This issue is valid

**Comment by hyoban** [comment]
We already support it, you can try it

![Image](https://github.com/user-attachments/assets/84e2235e-bd47-4723-b060-899f551ae56f)

**Comment by luo-chuan** [comment]
@hyoban I cannot type any word in this page.
![Image](https://github.com/user-attachments/assets/6c9c78de-fba9-4b8e-92a7-336c27a7dc85)

**Comment by hyoban** [comment]
https://github.com/user-attachments/assets/b4eaed75-0047-4709-93e5-47e567601433

Unable to reproduce, please provide more information

**Comment by luo-chuan** [comment]
> Unable to reproduce, please provide more information

I cannot type any word in the box when I use Firefox. 
The Edge broswer does not have this problem.

**Comment by luo-chuan** [comment]
Custom CSS does not work in mobile phone, like android Edge and Lemur browser.

**Comment by Innei** [comment]
Fixed in 58249210567c00a6e56b129cebe7cb319555347a
